### ISSUE TITLE
Change GA to GTM

### ISSUE THREAD

**Issue by Rulox** [moxie-lean/lean-theme#76]
For error tracking, analytics, etc.

**Comment by Rulox** [comment]
Should we have a custom field for the key @mitogh  ? OR just rely on the developer to add an acf or metafield for it?

**Comment by mitogh** [comment]
I guess a field should be enough, however the current GA is used to keep track of the JS errors only. @Rulox

**Comment by Rulox** [comment]
Yeah I saw it, but I mean so it's easier to add more things like analytics

**Comment by mitogh** [comment]
Yeah I guess we have an open HTML field that inserts data into the footer.
